models/system/bert-base-uncased/asset.yaml

```diff
@@ -1,3 +1,4 @@
 extra_config: model.yaml
 spec: spec.yaml
 type: model
+categories: ["Models"]
```

Comment:
I don't think applying the "Models" category to all of the models in this PR makes sense. Maybe "Hugging Face" would be worth considering, as well as "Language" or other things that could be used to group bert-base-cased and similar models.

Reply:
will do this next pr

Reply:
Would need help from @ManojBableshwar @swatig007 here. 

We will push "Models" to start with and change as necessary.